.github/workflows/bri-3.yml

```diff
@@ -0,0 +1,55 @@
+name: bri-3
+
+on:
+  push:
+    paths:
+      - examples/bri-3/**
+    branches:
+      - main
+  pull_request:
+    paths:
+      - examples/bri-3/**
+    branches:
+      - main
```

Comment:
this means it executes on all PRs to main?

Reply:
on all PRs on main that change files in bri-3 folder, i opened PR on my fork (url in PR description) that modifies other files and actions were not executed
path regex is set both on workflow level (paths), and on job level (working-directory)